utils/utils.go

```diff
@@ -19,7 +20,15 @@ func OpenInBrowser(url string) error {
 	if err != nil {
 		return err
 	}
-	return run.PrepareCmd(browseCmd).Run()
+	err = run.PrepareCmd(browseCmd).Run()
+	if err != nil {
+		fmt.Printf("Error opening %s in your browser.\n", url)
+		envVar := os.Getenv("BROWSER")
```

Comment:
This works, but what feels off to me about this approach is that now `utils` also has to know about the BROWSER environment variable, which was previously scoped to only the `browser` package. Maybe the latter package should expose a function called `BrowserEnv()` which we can query to see whether there was a user setting? This is the best suggestion I could come up with right now.

Reply:
One of my linters suggested that `browser.BrowserEnv` "stutters," so I went with `browser.FromEnv`.